.github/workflows/pr-validation.yml

```diff
@@ -27,11 +27,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
+        pip install setuptools wheel tox pytest
         pip install -r requirements.txt
     - name: Lint with flake8
       run: |
-        flake8 . --count --show-source --statistics --exit-zero
+        tox -e flake8
```

Comment:
Why do we need to replace the `flake8` command with a `tox` command?

Reply:
Ohh, I had updated the PR but forgot to comment here. 
As discussed, have updated with an if condition to skip linting check for 3.7 ver. 
And, build is green now.
Please check.